documentation/en/cli-lotus.md

```diff
@@ -7,7 +7,7 @@ USAGE:
    lotus [global options] command [command options] [arguments...]
 
 VERSION:
-   1.25.2
+   1.25.3-dev
```

Comment:
This is matching master, probably got here due to conflicts resolving; Before this gets into any staging branch this will change to v1.26.x-x, but for now anything -dev is more correct.